mmdet3d/visualization/local_visualizer.py

```diff
@@ -723,9 +731,10 @@ def add_datasample(self,
 
         if draw_gt and data_sample is not None:
             if 'gt_instances_3d' in data_sample:
-                gt_data_3d = self._draw_instances_3d(
-                    data_input, data_sample.gt_instances_3d,
-                    data_sample.metainfo, vis_task, palette)
+                if len(data_sample.gt_instances_3d) > 0:
```

Comment:
This has been fixed in https://github.com/open-mmlab/mmdetection3d/pull/2231, we do not need to modify this part